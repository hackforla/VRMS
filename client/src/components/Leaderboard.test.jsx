import React from "react";
import LeaderBoard from "./Leaderboard";
import renderer from "react-test-renderer";
import { test, expect } from 'vitest'

test("renders without crashing", () => {
  const tree = renderer.create(<LeaderBoard />).toJSON();
  expect(tree).toMatchSnapshot();
});
