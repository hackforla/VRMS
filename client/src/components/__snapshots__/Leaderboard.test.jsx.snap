// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders without crashing 1`] = `
<div
  className="flexcenter-container"
>
  <div
    className="dashboard-header"
  >
    <p
      className="dashboard-header-text-large"
    >
      Volunteer Leaderboard
    </p>
  </div>
  <div
    className="events-list"
  >
    <ul />
  </div>
</div>
`;
